### DISCUSSION TITLE
Some applications with their own GUI causes erratic mouse movements.

### DISCUSSION THREAD

**Discussion by fauxlefam** [rustdesk/rustdesk#9867] [General]
### Bug Description

The mouse pointer/cursor will move from X/Y to the very top-left of my screen while the applications is open. while I am still able move and click its very laggy and it takes a while to move the cursor from point A to point B.

### How to Reproduce

Please download the [FiveM](https://fivem.net/) client from CFX. and run the application. once you start moving your mouse you will notice the bug. 
Note that a valid installation of Grand Theft Auto: V is required to install FiveM.

### Expected Behavior

Normal mouse movements as per user-input.

### Operating system(s) on local side and remote side

Windows 10 => Windows 10

### RustDesk Version(s) on local side and remote side

1.3.2 => 1.3.2

### Screenshots

https://github.com/user-attachments/assets/fdc1b967-b295-4926-8db6-84f96e587fb6

### Additional Context

So while using FiveM which happen to be the only application I use for my remote work, the mouse pointer will move in such an erratic way where I can not control the mouse in real-time. instead I need to wait for the animation to finish before I can click or move.

**Comment by evatioc** [comment]
Did you solve this?

**Comment by rochera87** [comment]
I created there. I hope can help you:

https://gist.github.com/rochera87/8df477810fdabd5bdd45670e5f2199c3
